config/config-package.py

```diff
@@ -363,6 +365,23 @@ def tox(self):
             **options
         )
 
+    def _detect_robotframework(self):
+        """Dynamically find out if robotframework is used in the package.
+
+        We look at the dependencies, as we expect the depenedency checker
+        to make it easy for us.
+        """
+        try_files = (
+            self.path / 'setup.py',
```

Comment:
Don't we need `setup.cfg` here as well?
But we do not often use it for specifying dependencies.  And we are more likely to move everything over to `pyproject.toml`.  So it may be fine.